WorkloadManagementSystem/Service/PilotAgentsHandler.py

```diff
@@ -0,0 +1,50 @@
+""" Hello Service is an example of how to build services in the DIRAC framework
+"""
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB import PilotAgentsDB
+
+__RCSID__ = "$Id: $"
+
+import types
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Utilities import Time
+# from DIRAC.WorkloadManagementSystem.DB import PilotAgentsDB
+
+class PilotAgentsHandler( RequestHandler ):
```

Comment:
General comment: WMSAdministratorHandler interacts with PilotAgentsDB too. I am ok for having a handler for interacting with pilotLogging only. But I would like other reviewers to express their opinion on this point.